### PR TITLE
Update factstore deployment template 

### PR DESCRIPTION
**Pull request by 4n4nd**
add deployment commands in Makefile for sql and factstore deployment

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,3 +11,15 @@ run-with-local:
...
+	oc new-app centos/mysql-56-centos7 -e MYSQL_DATABASE=factstore -e MYSQL_PASSWORD=password -e MYSQL_USER=admin -e MYSQL_ROOT_PASSWORD=password
+
+oc_delete_sql_db:
+	oc delete all -l app=mysql-56-centos7
+
+oc_deploy_factstore:
+	oc process -f openshift/factstore.app.yaml | oc apply -f -
+
+oc_delete_factstore:
+	oc process -f openshift/factstore.app.yaml | oc delete -f -
```
Comment (zmhassan) [inline Makefile]: I like this quick commands.

File: openshift/factstore.app.yaml [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
+
 objects:
+
   - apiVersion: image.openshift.io/v1
     kind: ImageStream
     metadata:
       labels:
-        app: log-anomaly-detector-factstore-${APP_NAME}
+        app: log-anomaly-detector-factstore-${CUSTOMER_ID}
       name: log-anomaly-detector-factstore
```
Comment (zmhassan) [inline openshift/factstore.app.yaml]: Good idea fixing custom app_Names

File: openshift/factstore.app.yaml [hunk 3]
```diff
@@ -92,22 +97,25 @@ objects:
...
-          port: 5001
-          targetPort: 5001
+
 parameters:
   - name: SQL_CONNECT
-    value: mysql+pymysql://admin:admin@mysql-56-centos7/factstore
-  - name: APP_NAME
+    value: >
+      mysql+pymysql://admin:password@$(MYSQL_56_CENTOS7_SERVICE_HOST):$(MYSQL_56_CENTOS7_SERVICE_PORT)/factstore
+  - name: CUSTOMER_ID
```
Comment (zmhassan) [inline openshift/factstore.app.yaml]: @4n4nd Where are you getting `$MYSQL_56_CENTOS7_SERVICE_HOST` or `$MYSQL_56_CENTOS7_SERVICE_PORT` from ?

Comment (zmhassan) [reply openshift/factstore.app.yaml]: @4n4nd Perhaps it would be easier to revert this back to `mysql+pymysql://admin:admin@mysql-56-centos7/factstore`

Comment (4n4nd) [reply openshift/factstore.app.yaml]: its the service created when you deploy the sql db

Comment (4n4nd) [reply openshift/factstore.app.yaml]: You need this to connect to the external sqldb

Comment (4n4nd) [reply openshift/factstore.app.yaml]: @zmhassan

Comment (zmhassan) [reply openshift/factstore.app.yaml]: Okay then that would be fine.

**Pull request merged**